Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
zio-pietro <notifications@github.com> writes:
> @dmitrizagidulin  yes, I'm slowly discovering the background
> (personally I think it's a kind of NIH syndrom),


Most of Pillar's implementation and syntax (formerly known as Pier,
formerly known as Smallwiki, formerly known as Swiki) probably predates
all markups you know. So I can happily say that other markups have a
kind of NIH syndrome :-). But I agree that Pillar was (and is still) not
accessible to non-Smalltalk developers so it doesn't count.

> but I'll do my best of helping us to do good UPBE from typographics
> point of view even with Pillar. Still remain of opinion for short
> texts any markup language, for big works LaTeX. "Here I stand. I can
> do no other" [two hours long continuation deleted, I know, it's a lost
> battle, today's people want quick results with minimal effort and
> so-so quality is good enough]


I agree that LaTeX would allow best quality. But I think that
EnterprisePharo is already very good for something generated out of a
markup language. And we have HTML too.

-- 
Damien Cassou
http://damiencassou.seasidehosting.st

"Success is the ability to go from one failure to another without
losing enthusiasm." --Winston Churchill